### PR TITLE
Address broken rendering issues

### LINKED ISSUE #2465: Address broken rendering issues
Check for broken includes similar to https://github.com/dotnet/docs/pull/2464.

capture

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #2465

### CODE REVIEW

File: docs/core/tutorials/libraries.md [hunk 1]
```diff
@@ -132,8 +132,8 @@ Your project file could look like this:
 You'll notice three major changes here:
 
 1. The `TargetFramework` node has been replaced by `TargetFrameworks`, and three TFMs are expressed inside.
-2. There is an `<ItemGroup>` node for the `net40 ` target pulling in one .NET Framework references.
-3. There is an `<ItemGroup>` node for the `net45` target pulling in two .NET Framework references.
+1. There is an `<ItemGroup>` node for the `net40 ` target pulling in one .NET Framework references.
+1. There is an `<ItemGroup>` node for the `net45` target pulling in two .NET Framework references.
 
 The build system is aware of the following preprocessor symbols used in `#if` directives:
```
Comment (mairaw) [inline docs/core/tutorials/libraries.md]: perhaps something to address on a different PR, but the list below should probably have .NET Framework 4.7 and .NET Standard 2.0. (haven't tested though)

File: docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md [hunk 2]
```diff
@@ -15,39 +15,50 @@ author: "Erikre"
...
-### Parent Elements  
-  
-|Element|Description|  
-|-------------|-----------------|  
-|[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages, as well as routing tables that define the target endpoints to send messages to when a filter matches.|  
-  
-## See Also  
- <xref:System.ServiceModel.Routing.Configuration.FilterElement?displayProperty=fullName>    
+
+Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages.
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md]: missing space between include and API link

File: docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md [hunk 3]
```diff
@@ -15,39 +15,50 @@ author: "Erikre"
...
+
+**\<system.serviceModel>**
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md]: nice to have would be to link to each node

File: docs/framework/configure-apps/file-schema/wcf/routing.md [hunk 4]
```diff
@@ -15,39 +15,59 @@ author: "Erikre"
...
-### Parent Elements  
-  
-|Element|Description|  
-|-------------|-----------------|  
-|system.ServiceModel|The root element of all WCF configuration elements.|  
-  
-## See Also  
- <xref:System.ServiceModel.Routing.Configuration.RoutingSection?displayProperty=fullName>    
+
+Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages, as well as routing tables that define the target endpoints to send messages to when a filter matches.
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/routing.md]: missing space between include and api link

File: docs/framework/wpf/controls/index.md [hunk 5]
```diff
@@ -18,15 +18,14 @@ ms.author: dotnetcontent
...
+
 <a name="creating_an_instance_of_a_control"></a>   
 ## Creating an Instance of a Control  
  You can add a control to an application by using either                  [!INCLUDE[TLA#tla_xaml](../../../../includes/tlasharptla-xaml-md.md)] or code.  The following example shows how to create a simple application that asks a user for their first and last name.  This example creates six controls: two labels, two text boxes, and two buttons, in                  [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharptla-xaml-md.md)]. All controls can be created similarly.  
   
- [!code-xml[ControlsOverview#1](../../../../samples/snippets/csharp/VS_Snippets_Wpf/ControlsOverview/CSharp/Window1.xaml#1)]  
+ [!code-xaml[ControlsOverview#1](../../../../samples/snippets/csharp/VS_Snippets_Wpf/ControlsOverview/CSharp/Window1.xaml#1)]
```
Comment (mairaw) [inline docs/framework/wpf/controls/index.md]: interesting... this is new. it wasn't supported before AFAIK

**Review by mairaw — APPROVED** [review]
Oh my! There were some pretty bad broken topics in there. LGTM apart from a few nit comments.

**Comment by GuardRex** [comment]
@mairaw Sure thing. I'll get on this after the current priority work is squared away.

File: docs/core/tutorials/libraries.md (hunk 1 above)
Comment (GuardRex) [inline docs/core/tutorials/libraries.md]: Opened at https://github.com/dotnet/docs/issues/2561

File: docs/framework/wpf/controls/index.md (hunk 5 above)
Comment (GuardRex) [inline docs/framework/wpf/controls/index.md]: @mairaw It isn't supported; they recommend following http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases 

... BUT ... 

https://review.docs.microsoft.com/en-us/dotnet/framework/wpf/controls/index?branch=pr-en-us-2469

:grinning: **RexHacks:tm:** :grinning:

capture

Comment (mairaw) [inline docs/framework/wpf/controls/index.md]: I mean, this is a new addition to our custom extension. I've asked for it but haven't seen the bug resolved back to me. This should probably cause us to do a data fix to the content that was already migrated to converted from xml to xaml. I need to check to be sure.

Comment (GuardRex) [inline docs/framework/wpf/controls/index.md]: Rats! 🐀 I thought I hacked it. If you need an issue and a script run, I know an old green dinosaur who can knock that up. AFAICT all we need to do is to run it on the presence of the file extension (*.xaml*) and update the ident. Let me know if/when you want this done.

**Pull request merged**